CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ cmake_minimum_required (VERSION 3.8)
 
 project (XbSymbolDatabase)
 
+add_compile_definitions(_CRT_SECURE_NO_WARNINGS)
```

Comment:
This feels like it should only apply for MSVC

Reply:
XbSymbolCacheGenTest has this defined keyword. It's the only way to allow compile with MSVC. Somehow MSVC didn't check XbSymbolDatabase inspection for any defined keyword.

Source: https://github.com/Cxbx-Reloaded/XbSymbolCacheGenTest/blob/master/CMakeLists.txt

Reply:
Also, I'm only using MSVC since I am mainly Windows user btw. I cannot vouch for other compiler may have same issue or not.